### ISSUE TITLE
Move from OpenSubtitle to Addic7ed when we can

### ISSUE THREAD

**Issue by aloisdg** [exebetche/vlsub#97]
## Why ?

OpenSubtitle leech contents from addic7ed community and add ads.
They propose to register as VIP member to remove ads.

## How ?

If they read contents from their website, we could to.
There are repo on [github](https://github.com/search?utf8=%E2%9C%93&q=addic7ed)

If you are looking for others providers, there is a thread about this in [PopcornTime's forum](https://discuss.popcorntime.io/t/subtitles-for-tv-shows-from-addic7ed/2495/21).

This issue is kind of relative to issue #88.

**Comment by exebetche** [comment]
Yes, add multiple sources for subtitles is the most required feature. In fact I started working on it before Vlc 2.1 came out but I never finished, since I wasn't sure the extension will work again.
I don't plan to "move" to an other source, just add the possibility to add script to search and download from multiple websites, because opensubtitles offers multiple advantages, like a huge database of movie hash and release names that allow to get subs with right sync in just one click.
About their decision to put adds in the subtitles, I wouldn't be to hard on them. Yes, people grab the subtitles from any sources or community and upload them to OS, but again you can find people copy-pasting things on any content sharing website on the web. And I don't think they really put adds to make money, if you look at their stats¹, you can see they have up to 4 millions subtitles request on weekend days. That plus the requests to search the subtitles, the ddos that came with the notoriety... no free website can cope with this kind of load without a little commercial support.
¹http://www.opensubtitles.org/en/statistics

**Comment by aloisdg** [comment]
Thank you for the answer.

I will wait for the "multi sources" feature :)

> About their decision to put ads in the subtitles, I wouldn't be too hard on them.

I have my idea to deal with it. I can understand their move but it is very annoying.

**Comment by GAL18H** [comment]
> Yes, adding multiple sources for subtitles is the most required feature

is there any progress with the feature " multi sources"  ??

**Comment by bhankas** [comment]
Since Opensubtitles has gone on and decided to make this requirement pretty much a matter of time, Any update? The repo itself has not been added in a while, so we might want to start looking into forking things.

Source: https://blog.opensubtitles.com/opensubtitles/saying-goodbye-to-opensubtitles-org-api-embrace-the-20-black-friday-treat
